app/assets/javascripts/modules/remove-filter.js

```diff
@@ -0,0 +1,56 @@
+window.GOVUK = window.GOVUK || {}
```

Comment:
Nitpick: there's a lot of lines in this file without a semi colon at the end.

Reply:
I had a feeling we didn't use semi-colons in our JS? But I agree we should probably pick one or the other in this file.